### ISSUE TITLE
Streamline Master Agreement workflow in Knack Purchasing 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#4495]
> What application are you using?

Finance & Purchasing

> Describe the problem.

In Knack Purchasing we have cases where we are tracking the creation of a Master Agreement.  Through this process we are experiencing getting the PR to a closed status without jumping through a lot of hoops.  Typically the PR will go to an AP status where it is supposed to have invoices assigned and paid against the balance.  We'd like to be able to take a status to closed for items such as this.

> Describe the outcome you'd like to see when this feature is implemented.

Our team will be able to close PRs and keep our system "clean" more efficiently and effectively through the implementation of a closed status option for PRs.

> Describe any workarounds you currently have in place or alternative solutions you've considered.

Currently our option is to move the PR to "processing" then "invoicing" then to "closed"

> Requested By
Dan V.

Request ID: DTS20-101330

**Comment by johnclary** [comment]
Schedule meeting w/ Dan for 12/7.

**Comment by johnclary** [comment]
Per discussion, I have updated the PR status workflow so that the RQM (Request for Master Agreement) will now move to a status of "Closed" once "PURCHASE_PROCESSING_COMPLETE" is yes. This bypasses the accounts-payable side of things. These RQMs are not actually purchases, they are merely documenting the setup of a new MA against which future purchases will be billed.

I also added the "Notes to purchasing" field to the advanced search page, which came up during discussion.
